__tests__/data/am_output.json

```diff
@@ -3593,5 +3593,113 @@
     },
     "files": {},
     "userId": "my-anonymous-id-new"
+  },
+  {
+    "version": "1",
+    "type": "REST",
+    "method": "POST",
+    "endpoint": "https://api.amplitude.com/2/httpapi",
+    "headers": {
+        "Content-Type": "application/json"
+    },
+    "params": {},
+    "body": {
+        "JSON": {
+            "api_key": "abcde",
+            "events": [
+                {
+                    "os_name": "Android",
+                    "os_version": "7.1.1",
+                    "device_model": "Android SDK built for x86",
+                    "device_manufacturer": "Google",
+                    "device_id": "5d205961641ee6c5",
+                    "carrier": "Android",
+                    "app_name": "Sample Kotlin",
+                    "app_version": "1.2",
+                    "platform": "Android",
+                    "language": "en-US",
+                    "event_properties": {
+                        "name": "Screen event"
+                    },
+                    "insert_id": "1662393883248-509420bf-b812-4f8d-bdb2-8c811bfde87f",
+                    "user_properties": {
+                        "anonymousId": "5d205961641ee6c5",
+                        "id": "User Android",
+                        "userId": "User Android"
+                    },
+                    "event_type": "Viewed Screen event Screen",
+                    "user_id": "User Android",
+                    "device_brand": "Google",
+                    "time": 1662393883250,
+                    "session_id": "1662393792",
```

Comment:
This is Long defined by Amplitude docs. We are sending string
![Screenshot 2022-09-15 at 1 14 29 PM](https://user-images.githubusercontent.com/25389167/190345654-0ab81f54-4dc3-48cc-a83c-45606ad5e878.png)

Reply:
@utsabc , we are handling it now, can you please take a look again !

Reply:
Sure